specification/http/src/main/scripts/org/kaazing/specification/http/rfc7235/framework/invalid.then.valid.credentials/request.rpt

```diff
@@ -0,0 +1,25 @@
+property invalidCredentials ${http:loginBase64Encoder("baduser:badpass")}
+property authHeader1 ${http:append("Basic ", invalidCredentials)}

```

Comment:
rather than naming authHeader1/authHeader2, the name should reflect the purpose as much as possible
authHeader1 can be invalidAuthValue
authHeader2 can be validAuthValue